runhttpd.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
+
+rmdir /var/www/html
+ln -s /shared/html/ /var/www/html
```

Comment:
nit: this could probably go to the httpd configuration

Reply:
Good point, I'll change this in follow-on patch that will change the http port we're using.